src/js/selfVoicing.js

```diff
@@ -16,19 +16,11 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
     fluid.registerNamespace("gpii.firstDiscovery");
 
     fluid.defaults("gpii.firstDiscovery.selfVoicing", {
-        gradeNames: ["gpii.firstDiscovery.attachTooltip", "fluid.textToSpeech", "autoInit"],
+        gradeNames: ["gpii.firstDiscovery.attachTooltip", "fluid.prefs.msgLookup", "fluid.textToSpeech", "autoInit"],
```

Comment:
The navButtons component probably should have "fluid.prefs.msgLookup" grade added too so we can use a common message resolving strategy in all non-prefEditor components.

Thinking furthur, we can create a new grade component that contains: this grade, the subcomponent "msgResolver" and the distributeOptions for distributing messageBase. This new grade can be applied to all non-prefEditor components.